piwebapi_samples/CommonActions/README.md

```diff
@@ -0,0 +1,76 @@
+# PI Web API Samples
```

Comment:
`CommonActions` as a folder name is ok, but should we consider something like `API`, or `REST_API`? It's really a walkthrough of the API in different languages. `APITour`? I guess my instinct is just that `API` should be in the folder name in one way or another.

Reply:
@derekendres don't forget to change the settings on the PI Web API pipelines in ADO to point to the new folder location. Right now they're pointing to a different location.